third_party/nvfuser/csrc/ir_interface_nodes.h

```diff
@@ -655,7 +657,8 @@ class TORCH_CUDA_CU_API TensorViewBuilder {
  private:
   size_t ndims_ = 0;
   DataType dtype_ = DataType::Float;
-  std::vector<bool> contiguity_;
+  std::vector<c10::optional<bool>> contiguity_;
+  c10::optional<bool> uniform_contiguity_ = c10::nullopt;
```

Comment:
Can you leave a short comment here about, e.g., what the uniform contiguity means, what happens if both are given, etc.

Reply:
added